tests/test_basic_functionality.py

```diff
@@ -458,18 +466,22 @@ def test_to_dict_yaml(test_spec: Path) -> None:
     assert yaml_config == lct.to_dict()
 
 
-def test_getter() -> None:
-    getter_dict: NestedDict = {
-        "outer_layer_1": "test_value",
-        "outer_layer_2": {"inner_layer": "test_value2"},
-    }
+def test_getter(getter_dict: NestedDict) -> None:
     lct = LayeredConfigTree(getter_dict)
 
     assert lct.get("outer_layer_1") == "test_value"
     assert lct.get("outer_layer_2").to_dict() == getter_dict["outer_layer_2"]  # type: ignore [union-attr]
     assert lct.get("outer_layer_2").get("inner_layer") == "test_value2"  # type: ignore [union-attr]
 
 
+def test_tree_getter(getter_dict: NestedDict) -> None:
+    lct = LayeredConfigTree(getter_dict)
+
+    assert lct.get_tree("outer_layer_2").to_dict() == getter_dict["outer_layer_2"]
+    with pytest.raises(ValueError, match="not a LayeredConfigTree"):
+        assert lct.get_tree("outer_layer_1") == "test_value"
```

Comment:
Should add a case where you try to get a key that doesn't exist and confirm it throws the right errors.

I think we should remove the default argument, but if we were to keep it, we ought to have tests of that as well.